client/sections/edit/edit.section.ng.jade

```diff
@@ -0,0 +1,17 @@
+md-whiteframe.topic-form.md-whiteframe-z1(layout="column", layout-align="center center")
+    .title
+        h2 Edit section '{{ edit.section.name }}'
+    form(ng-submit="sectionCtrl.section.update(edit.section)", layout="column")
+        md-input-container
+            input(type="text" ng-model="edit.section.name"
+                    placeholder="Section name" aria-label="section name")
+        md-input-container
+            input(type="description" ng-model="edit.section.description"
+                    placeholder="description" aria-label="Description")
```

Comment:
Tu to samo :) Chyba, że tam jest "caps lock" włączony na placeholder, bo w sumie nie pamiętam